### PR TITLE
Compare pb and physical benchmark

### PR DESCRIPTION
**Pull request by arthbous**
* Compare PB and PNP works perfectly.
* Still need a test for PB+NS
* physic_bench contains the physical benchmark. The PNP one works fine, the only problem is overrifinement.

### CODE REVIEW

**Comment by arthbous** [comment]
It's working!!

The magnitude of the velocity increases when the voltage drop increases. I will do more tests to see what is the max voltage drop the code can handle.

@maximilianmetti , I am thinking of doing the Na+Cl physical constants, what do you think?
@maximilianmetti , you can do some tests on just PNP with small epsilons and EAFE.

**Comment by maximilianmetti** [comment]
I think Na+Cl in water should be good. Maybe set the reference density to that of the PNP + Steric Effects paper and have the length scale be 100nm?

I'll pull this branch and test with small epsilon too. (Equivalently, setting the scale to be 10microns or something.)

**Comment by arthbous** [comment]
// For L = 10 nm, T = 298K
  std::map<std::string, std::vector<double>> coefficients = {
    {"permittivity", {0.019044}},
    {"diffusivity0", {1.0}},
    {"diffusivity1", {1.334/2.032}},
    {"valency0", {1.0}},
    {"valency1", {-1.0}},}

But that when the box is 1x1x1, when we do PNP the box is 10x1x1 does that mean we should deviede Epsilon by 10 when we do PNP?

**Comment by maximilianmetti** [comment]
Just change the size of the box. L should be the same and that means that eps is unchanged.

File: benchmarks/physic_bench/pnp_newton_solver.h [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+  double Eps = 1E-3;
+  printf("Initialize coefficients\n");
+  std::map<std::string, std::vector<double>> pnp_coefficients = {
+    {"permittivity", {Eps}},
+    {"diffusivity", {0.0, 1.0, 1.0}},
+    {"valency", {0.0, 1.0, -1.0}}
+  };
+  std::map<std::string, std::vector<double>> pnp_sources = {
+    {"fixed_charge", {0.0}},
+    {"g", {10.0*Eps}}
```
Comment (maximilianmetti) [inline benchmarks/physic_bench/pnp_newton_solver.h]: what is this condition? i don't the `Eps` is correct

**Pull request merged**